batch7crud-roll2/src/main/webapp/css/style.css

```diff
@@ -0,0 +1,305 @@
+    width: 100%;
+    float: left;
+    margin-bottom: 10px;
+    height: 30px;
+}
+
+.loginBox input[type=submit] {
+    clear: both;
+    float: right;
+    width: 50%;
+    margin-top: 5px;
+    font-family: montserrat, sans-serif;
+    height: 30px;
+}
+
+.errorMessage {
+    font-size: 12px;
+    color: #c9302c;
+}
+
+/*Left Navigation*/
+
+.left ul li {
+    border-bottom: 1px solid #336666;
+    cursor: pointer;
+}
+
+.left ul li a {
+    width: 100%;
+    font-size: 16px;
+    display: block;
+    color: #ffffff;
+    text-align: center;
+    line-height: 40px;
+}
+
+/*List Page*/
+
+.bodyTitle {
+    text-align: left;
+    width: 97%;
+    padding: 20px 0 20px 3%;
+    font-size: 18px;
+    font-weight: bold;
+    background-color: #eeeeee;
+    border-bottom: 2px solid #333333;
+}
+
+.loggedinUser {
+    font-size: 12px;
+    float: right;
+}
+
+.logout {
+    clear: both;
+    float: right;
+}
+
+.addNewEmployee {
+    padding: 20px 0 20px 3%;
+}
+
+.addNewEmployee a {
+    width: 10%;
+    background-color: #336699;
+    color: #ffffff;
+    display: block;
+    font-size: 14px;
+    text-decoration: none;
+    text-align: center;
+    line-height: 30px;
+    padding-left: 1%;
+    padding-right: 1%;
+}
+
+.employeeListTable {
+    padding-left: 3%;
+}
+
+.employeeListTable table {
+    border: 1px solid #cccccc;
+    width: 95%;
+}
+
+.employeeListTable table tr {
+    line-height: 50px;
+    font-size: 14px;
+    color: #666666;
+}
+
+.employeeListTable table tr:nth-child(odd) {
+    background-color: #eeeeee;
+}
+
+.employeeListTable table td {
+    padding-left: 5px;
+    border-right: 2px solid #cccccc;
+    width: 15%;
+    padding-right: 5px;
+}
+
+.employeeListTable table td:first-child {
+    width: 10%;
+}
+
+.employeeListTable table td:nth-child(n+6) {
+    width: 5%;
+    border-right: none;
+}
+
+.employeeListTable table td a {
+    width: 100%;
+    display: block;
+    background: #336699;
+    color: #ffffff;
+    text-align: center;
+    line-height: 30px;
+}
+
+.employeeListTable table td:nth-last-child(3) a {
+    background: #5cb85c;
+}
+
+.employeeListTable table td:last-child a {
+    background: #c9302c;
+}
+
+.pagination {
+    float: right;
+    padding: 5px 4.8% 5px 0;
+}
+
+.pagination ul {
+    border: 1px solid #cccccc;
+    line-height: 40px;
+    font-size: 14px;
+    background: #eeeeee;
+}
+
+.pagination ul li {
+    display: inline-block;
+    border-right: 1px solid #cccccc;
+    padding: 0 5px;
+}
+
+.pagination ul li:last-child {
+    border: none;
+}
+
+.pagination ul li a {
+    width: 100%;
+    display: block;
+    color: #666666;
+    text-align: center;
+    line-height: 40px;
+}
+
+/*View Employee*/
+.viewEmployee {
+    padding: 10px 0 10px 3%;
+}
+
+.viewEmployee ul {
+    width: 40%;
+}
+
+.viewEmployee ul li {
+    display: block;
+    padding: 5px 5px;
+    color: #ffffff;
+    background: #336699;
+    border: 1px solid #6b8fa3;
+}
+
+.viewEmployee ul li span {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename selector viewEmployee to match the regular expression: ^[a-z][a-z\-]*$ [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=css%3Aselector-naming-convention)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Simplify this over-specified selector [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=css%3Aoverspecific-selectors)